src/PiWeb.Import.Sdk/ImportFiles/IImportGroup.cs

```diff
@@ -28,37 +27,37 @@ public interface IImportGroup
     /// <summary>
     /// The primary file with which the grouping was started. This file has to be recognized by the format.
     /// </summary>
-    IFileSource PrimaryFile { get; }
+    IImportFile PrimaryFile { get; }
 
     /// <summary>
     /// The list of additional files recognized by the import format and added using <see cref="AddFile"/>.
     /// The list does not include <see cref="PrimaryFile"/>. These files will not be offered to other import formats.
     /// </summary>
-    ICollection<IFileSource> AdditionalFiles { get; }
+    IEnumerable<IImportFile> AdditionalFiles { get; }

```

Comment:
I would suggest to use either `IReadOnlyCollection` or `IReadOnlyList` here, since those are far more convenient.

Reply:
OK, IReadOnlyList should be fine. But then I will remove AllFiles as don't want to allocate and maintain two collections just for that.